icu-date.lua

```diff
@@ -191,6 +191,21 @@ function _M:format(format)
   return uchar_to_string(result, result_length)
 end
 
+local check_length
+-- Workaround for https://unicode-org.atlassian.net/browse/ICU-21802
+-- See also https://github.com/tarantool/icu-date/issues/27
+local version = ffi.new('UVersionInfo')
+local ok = pcall(function() ffi.C.u_getVersion(version) end)
```

Comment:
Just curious: in what circumstances an error can be raised here?

Reply:
When statically built Tarantool doesn't export u_getVersion symbol. E.g. it's relevant for all Tarantool EE versions before https://github.com/tarantool/tarantool-ee/commit/c254bb1889dc2f6aff65b5e11d08e669a9af11c9 
Just to avoid breaking something.

Reply:
Got it, thanks.